odin/assetlist.json

```diff
@@ -22,6 +22,27 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/odin.svg",
 	"png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/odin.png"
       }
-    }
+    },
+    {
+        "description": "GEO token for ODIN Protocol",
+        "denom_units": [
+          {
+            "denom": "mGEO",
+            "exponent": 0
+          },
+          {
+            "denom": "GEO",
+            "exponent": 6
+          }
+        ],
+        "base": "mGEO",
+        "name": "GEO",
+        "display": "geo",
+        "symbol": "GEO",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/geo.svg",
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/odin/images/geo.png"
```

Comment:
can we please make the png have square aspect ratio? The width can go right to the edge of the token--whereas the extra padding in the file can mess with how it's rendered on dApps.